src/styles/blocks/card.scss

```diff
@@ -0,0 +1,96 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 16px;
+  background: #ffff;
+  width: 200px;
+  margin-bottom: 46px;
+  margin-right: 48px;
```

Comment:
Pay attention to your CSS properties order.
https://9elements.com/css-rule-order/